docs/installing-northstar/troubleshooting.md

```diff
@@ -80,7 +80,7 @@ On newer Intel CPUs you might see a message like this
 If you are seeing this in the main menu of TF|2 and have a 10th or 11th generation Intel CPU this is a bug which has a simple fix:
 
 In the Windows Start menu on the bottom left search for "Edit the system environment variables" and open the program. In the "advanced" tab click on "Environment Variables..." near the bottom.\
-In System Variables (not user variables) click "New..." and add a new system variable where the variable name is `OPENSSL_ia32cap` and the value is `~0x200000200000000`. Make sure to click OK to apply the changes. Finally restart your device and you should be good to go.
+In System Variables (not user variables) click "New..." and add a new system variable where the variable name is `OPENSSL_ia32cap` and the value is `0x200000200000000`. Make sure to click OK to apply the changes. Finally restart your device and you should be good to go.
 
 If you're on Linux, you can set the appropriate environment variable via `env OPENSSL_ia32cap=~0x20000000 %command%`.
```

Comment:
```suggestion
If you're on Linux, you can set the appropriate environment variable via `env OPENSSL_ia32cap=0x20000000 %command%`.
```
Should the Linux equivalent not also be changed?

Reply:
I'll be honest I just didn't notice that part, I also have no idea if it matters for Linux, I'm going to assume so so I'll commit this